bots/lemmy_mlb_game_threads/__init__.py

```diff
@@ -6339,3 +6324,13 @@ def bot_state(self):
 
         self.log.debug("Bot Status: {}".format(botStatus))  # debug
         self.bot.detailedState = botStatus
+
+    def _truncate_post(self, text):
+        warning_text = '\n# Truncated: 10k Limit Reached'
+        max_length = self.settings["Lemmy"]["POST_CHARACTER_LIMIT"] - len(warning_text)
```

Comment:
What's the difference between this syntax and `self.settings.get("Lemmy", {}).get("POST_CHARACTER_LIMIT", "")`?

Reply:
.get has the option to specify a default, while this syntax will get a key error exception if the value doesn't exist.  While the value is mandatory and a default is provided in the config, we can change it to have a hard coded default in this case as well:

```suggestion
        max_length = self.settings.get("Lemmy", {}).get("POST_CHARACTER_LIMIT", 10000)
```

self.settings is a dict with the contents of the config.  This gets the "Lemmy" section of the config which is itself a dict, but will substitute an empty dict if there isn't a Lemmy key.  Then it looks up "POST_CHARACTER_LIMIT" in the dict.  If that key isn't in "Lemmy" or the empty dict, it will return 10000.  It would have the same effect as:

```suggestion
        try:
            max_length = self.settings["Lemmy"]["POST_CHARACTER_LIMIT"] - len(warning_text)
        except:
            max_length = 10000 - len(warning_text)
```


I don't like to using .get with a default if the code won't work properly with that default.

Reply:
I didn't want to hard code the max post length because it may change in the future.  My first suggested change makes the internal default 10k, but that can be overridden by the config without modifying the code.  This also allows for the possibility of different instances having different maximums.

Reply:
Ok, should be good now